### PR TITLE
feat: upgrade ethers.js v6   06-ethersjs-waffle

### PR DESCRIPTION
**Pull request by gongdaxia**
1. upgrade ethers.js v6;
2. update reademe.md

### CODE REVIEW

**Review by yingjingyang — APPROVED** [review]

**Review by 0x-stan — APPROVED** [review]

File: basic/06-ethersjs-waffle/README-cn.md [hunk 1]
```diff
@@ -92,7 +92,12 @@ $ yarn --versionyarn --version
...
-- 中文文档： <https://learnblockchain.cn/docs/ethers.js/api-providers.html>
+- ehterjs 官方文档： 
+  - v4 <https://docs.ethers.io/v4/api-providers.html>  
+  - v5 <https://docs.ethers.io/v5/getting-started/#getting-started--contracts>
+  - v6 <https://docs.ethers.org/v6/getting-started>
+
+- 中文文档： 
+  - v4 <https://learnblockchain.cn/docs/ethers.js/api-providers.html>
+  - v5 <https://learnblockchain.cn/ethers_v5/getting-started>
+  - v6 <https://www.wtf.academy/docs/ethers-101/HelloVitalik/>
```
Comment (stevending1st) [inline basic/06-ethersjs-waffle/README-cn.md:103]: It is recommended to put the high version in front and the lower version in the back.

```suggestion
- ehterjs 官方文档： 
  - v6 <https://docs.ethers.org/v6/getting-started>
  - v5 <https://docs.ethers.io/v5/getting-started/#getting-started--contracts>
  - v4 <https://docs.ethers.io/v4/api-providers.html>  

- 中文文档： 
  - v6 <https://www.wtf.academy/docs/ethers-101/HelloVitalik/>
  - v5 <https://learnblockchain.cn/ethers_v5/getting-started>
  - v4 <https://learnblockchain.cn/docs/ethers.js/api-providers.html>
```

File: basic/06-ethersjs-waffle/README.md [hunk 2]
```diff
@@ -93,7 +93,12 @@ afer changed:
...
+- etherjs offical document: 
+  - v4 <https://docs.ethers.io/v4/api-providers.html>  
+  - v5 <https://docs.ethers.io/v5/getting-started/#getting-started--contracts>
+  - v6 <https://docs.ethers.org/v6/getting-started>
 
-- Chinese document: <https://learnblockchain.cn/docs/ethers.js/api-providers.html>
+- Chinese document: 
+  - v4 <https://learnblockchain.cn/docs/ethers.js/api-providers.html>
+  - v5 <https://learnblockchain.cn/ethers_v5/getting-started>
+  - v6 <https://www.wtf.academy/docs/ethers-101/HelloVitalik/>
```
Comment (stevending1st) [inline basic/06-ethersjs-waffle/README.md:104]: It is recommended to put the high version in front and the lower version in the back.

File: basic/06-ethersjs-waffle/index.js [hunk 3]
```diff
@@ -6,22 +6,27 @@ const { ethers } = require("ethers");
 require("dotenv").config();
 const privateKey = process.env.PRIVATE_KEY;
 
-// const web3 = new Web3.providers.HttpProvider('https://sepolia.infura.io/v3/0aae8358bfe04803b8e75bb4755eaf07');
+// const web3 = new Web3.providers.HttpProvider('https://sepolia.infura.io/v3' + process.env.INFURA_ID);
```
Comment (stevending1st) [inline basic/06-ethersjs-waffle/index.js:9]:
```suggestion
// const web3 = new Web3.providers.HttpProvider(`https://sepolia.infura.io/v3${process.env.INFURA_ID}`);
```

**Comment by stevending1st** [comment]
@gongdaxia  Please review required.

**Review by stevending1st — APPROVED** [review]
LGTM

**Comment by gongdaxia** [comment]
@stevending1st ok, modify completed.

**Review by yingjingyang — APPROVED** [review]

**Pull request merged**